gobblin-api/src/main/java/org/apache/gobblin/runtime/api/SpecExecutor.java

```diff
@@ -23,29 +23,36 @@
 
 import com.typesafe.config.Config;
 
-import org.apache.gobblin.annotation.Alpha;
-
 
 /**
- * Defines a SpecExecutorInstance (typically a standalone instance, cluster or Azkaban deployment)
- * that can execute a {@link Spec}.
+ * Defines a representation of JobSpec-Executor in GaaS.
+ * A triplet of <Technology, location, communication mechanism> uniquely defines an object of SpecExecutor.
```

Comment:
What does 'Technology' term mean here? A simple example (maybe in the javadoc or unit test) would be helpful for others to understand the terminology.